app/models/performance_dashboards/overview.rb

```diff
@@ -126,4 +126,26 @@ def available_breakdowns
     breakdowns[:coc] = 'By CoC' if GrdaWarehouse::Config.get(:multi_coc_installation)
     breakdowns
   end
+
+  protected def filter_selected_data_for_chart(data)
+    labels = data.delete(:labels) || {}
+    chosen = data.delete(:chosen)&.to_set
+    chosen&.delete(:all)
+    if chosen.present?
+      (columns, categories) = data.values_at(:columns, :categories)
+      initial_categories = categories.dup
+      date = columns.shift
+      filtered = columns.zip(categories).select { |_, cat| cat.in?(chosen) }
+      data[:columns] = [date] + filtered.map(&:first)
+      data[:categories] = filtered.map(&:last)
+      excluded_categories = initial_categories - data[:categories]
+      if excluded_categories.present?
+        # FIXME: - pack this option into the columns so I don't have to modify 20+ calls in partials
```

Comment:
Can we just stick a method on the report that returns the appropriate excluded categories?  It would need to know the breakdown and which categories are included/excluded.  I think that could also be used in the view.

Reply:
@eanders Yes, I'm sure there are other ways this could be handled. Happy to discuss this with you if you have a few minutes